DigitalLearningSolutions.Data/Services/RegistrationService.cs

```diff
@@ -0,0 +1,69 @@
+﻿namespace DigitalLearningSolutions.Data.Services
+{
+    using System;
+    using DigitalLearningSolutions.Data.DataServices;
+    using DigitalLearningSolutions.Data.Models.Email;
+    using DigitalLearningSolutions.Data.Models.Register;
+    using MimeKit;
+
+    public interface IRegistrationService
+    {
+        string RegisterDelegate(DelegateRegistrationModel delegateRegistrationModel, string baseUrl);
+    }
+    public class RegistrationService: IRegistrationService

```

Comment:
NIP, but missing an empty line above this (the auto-formatter should get this, although not 100% about IntelliJ)